dbms/src/Flash/Coprocessor/DAGContext.h

```diff
@@ -374,6 +374,8 @@ class DAGContext
     /* const */ bool is_disaggregated_task = false; // a disagg task handling by the write node
     // `tunnel_set` is always set by `MPPTask` and is used later.
     MPPTunnelSetPtr tunnel_set;
+    // `mpp_receiver_set` is always set by `MPPTask` and is used later.
```

Comment:
https://github.com/pingcap/tiflash/pull/9073/files#diff-0878026e4933aaca8d4180924369d51711e37f6093444231bf1665481d93b068R186-R187
Because it needs to be public

Reply:
It can be solved by adding a public function?

Reply:
A public function can be added here, but since mpp-tunnel-set is already in the public section, it might be better to maintain consistency.

Reply:
But it will change the destructor order in `DAGContext`, I'm not sure if there is some potential risk of this.

Reply:
ok, updated.